src/test/resources/epub3/navigation-document.feature

```diff
@@ -10,5 +10,95 @@ Feature: EPUB 3 Navigation Document
         defined in the `navigation.feature` feature file.
 
   Background: 
-    Given EPUB test files located at '/epub3/files/epub/'
-    And EPUBCheck with default settings
+    Given EPUBCheck configured to check a Navigation Document
+    And test files located at '/epub3/files/navigation-document-xhtml/'
+
+  Scenario: Minimal nav document
```

Comment:
How about a common pattern for valid/invalid using something like verify/report.

So in this case, it might be: Verify a minimal navigation document.

While an invalid test case might then say something like "Report a navigation document without a toc".

(Sort of taking inspiration from schematron assertions, I suppose.)